.circleci/config.yml

```diff
@@ -36,23 +36,28 @@ job-references:
       - checkout
       - run: *setup_environment
       - run: *install_dependencies
+      - run:
```

Comment:
Wouldn't it make more sense to install these dependencies in the `install_dependencies` mixin?